File: Reactivities.Server/API/Models/Activities/ActivityViewModel.cs
using API.Models.Common;

namespace API.Models.Activities
{
    public class ActivityViewModel : BaseApiModel
    {
        public int? Id { get; set; }
        
        public string Title { get; set; }
        
        public string Date { get; set; }
        
        public string Description { get; set; }
        
        public string Category { get; set; }
        
        public string City { get; set; }
        
        public string Venue { get; set; }
    }
}

File: client/src/autogenerated/a-p-i/activities/models/activity-view-model.ts
import { BaseApiModel } from '../../common/base-api-model';

export class ActivityViewModel extends BaseApiModel {
    id?: number;
    title: string;
    date = new Date();
    description: string;
    categoryId: number;
    category: string;
    city: string;
    venue: string;
}
